chrome-launcher/README.md

```diff
@@ -97,3 +97,22 @@ chromeLauncher.launch({
   console.log(`Chrome debugging port running on ${chrome.port}`);
 });
 ```
+
+### Continuous Integration
+
+Unlike development environment, on CI environment Chrome binary is not present. 
```

Comment:
Rework of the paragraph: 

In a CI environment like Travis, Chrome may not be installed.  If you want to use `chrome-launcher`, you can install Chrome using Lighthouse's `download-chrome.sh` script:

    curl -L https://raw.githubusercontent.com/GoogleChrome/lighthouse/master/lighthouse-core/scripts/download-chrome.sh | bash

Then in `.travis.yml`, use it like so: